dosing_pump_controller/PumpController.h

```diff
@@ -82,6 +88,7 @@ class PumpController {
         uint8_t duty_cycle; // The current duty_cycle pump setting value, the PWM duty cycle the pump will operate at.
         uint8_t dose_frequency; // The current dose_frequency pump setting value, number of times per day the dose should be delivered.
         unsigned long dose_duration; // The current dose_duration pump setting value, how long each dose will last in milliseconds.
+        unsigned long interval_duration; // The length of time in seconds between each scheduled dose.
```

Comment:
Seconds or milliseconds? After confirming this let's double check the math in `pollPumpStatus()`

Reply:
Good callout, it is confusing switching to seconds and milliseconds. Let's just stick to milliseconds.

Reply:
Updated